### PR TITLE
New issue templates

### PR DESCRIPTION
**Pull request by behackl**
This PR improves the formatting of the existing issue template on GitHub and introduces three further templates: suggestions, feature requests, and installation issues.

## Motivation
Currently all issues filed are filed as bugs with ` [BUG-General]  ` in the title, and they are all flagged as bugs: we definitely need at least one further template for non-bug issues. Also, given the recent increase in the creation of issues, I feel it would let us keep the overview a bit easier if things were at least basically labeled correctly.

Personally, I think that classification of issues should happen via labels; duplicating the information will unavoidably lead to inconsistencies.

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/installation_issue.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+assignees: ''
+
+---
+
+#### Preliminaries
+
+- [ ] I have followed the latest version of the
+      [installation instructions](https://docs.manim.community/en/latest/installation.html).
+- [ ] I have checked the [troubleshooting problem]() and my problem is either not mentioned there,
+      or the solution given there does not help.
```
Comment (leotrs) [inline .github/ISSUE_TEMPLATE/installation_issue.md:15]:
```suggestion
- [ ] I have checked the [troubleshooting page](https://docs.manim.community/en/latest/installation/troubleshooting.html) and my problem is either not mentioned there,
      or the solution given there does not help.
```

**Review by leotrs — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/installation_issue.md (hunk 1 above)
Comment (behackl) [reply .github/ISSUE_TEMPLATE/installation_issue.md:15]: Well, seems my thoughts were somewhere else when I wrote that. Thanks for fixing it!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -32,47 +36,47 @@ Paste your code here.
 
 </details>
 
-**Logs**
-<details><summary>Terminal output (Screenshots acceptable)</summary>
+
+## Logs
+<details><summary>Terminal output</summary>
```
Comment (huguesdevimeux) [inline .github/ISSUE_TEMPLATE/bug_report.md:42]: **Suggestion** : precise here that logs should be generated using `-v DEBUG` flag

**Review by huguesdevimeux — APPROVED** [review]
SGTM :D

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (behackl) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good idea! 4d11443a

**Pull request merged**